semantic-segmentation-deeplabv3/module_specs/score.yaml

```diff
@@ -0,0 +1,69 @@
+name: Deeplabv3 Score
+id: 642e5483-b99b-4749-9568-ae48c8a98fa7
+category: Machine Learning\Score
+description: Loads a Deeplabv3 Net for semantic segmentation and inferences it
+inputs:
+- name: Model type
+  type: Mode
+  default: 'deeplabv3_resnet101'
+  description: model_type
+  options:
+    - 'deeplabv3_resnet101'
+    - 'fcn_resnet101'
+- name: Model path
+  type: ModelFolder
+  port: True
+  description: model_path
+- name: Data path
+  type: TabularDataFolder
+  port: True
+  description: data_path
+- name: Use CUDA
+  type: Boolean
+  default: False
+  description: use_cuda
+outputs:
+- name: Save path
+  type: TabularDataFolder
+  description: save_path
+implementation:
+  container:
+    runConfig:
+      baseDockerImage: mcr.microsoft.com/azureml/base-gpu:intelmpi2018.3-cuda10.0-cudnn7-ubuntu16.04
+      gpuSupport: True
+    conda:
+      name: project_environment
+      channels:
+      - defaults
+      dependencies:
+      - python=3.6.3
+      - pip:
+        - --extra-index-url=https://test.pypi.org/simple
+        - azureml-core==1.0.48
+        - alghost==0.0.78
+        - pandas==0.24.2
+        - opencv-python==4.1.0.25
+        - https://download.pytorch.org/whl/cu100/torch-1.1.0-cp36-cp36m-linux_x86_64.whl
+        - https://download.pytorch.org/whl/cu100/torchvision-0.3.0-cp36-cp36m-linux_x86_64.whl
+        - pillow==6.0.0
+        - pyarrow==0.14.0
+        - git+https://github.com/zyh911/CustomModules.git@t-yuhazh/add_semantic_segmentation_deeplabv3#subdirectory=semantic-segmentation-deeplabv3
+    command:
+    - python
+    - -m
+    - script.score
```

Comment:
Would be better to change package name 'script' because it looks improper for a package